### DISCUSSION TITLE
About next export & build

### DISCUSSION THREAD

**Discussion by infonexage** [nextauthjs/next-auth#4156] [Help]
### What is the improvement or update you wish to see?

Next Js Next Auth 4.2.1 working  fine using credentials provider in localhost. Once moving to production server after taken export & build for static chunk files  "api/auth/signin" page not found. Api folder is missing & not bundled with build. 
So need to update for this issue

Thanks

### Is there any context that might help us understand?

Next Export & Build

**Comment by ThangHuuVu** [comment]
Sounds like a configuration problem. This might help: https://github.com/nextauthjs/next-auth/discussions/2670
